lib/api_repo.js

```diff
@@ -430,6 +437,51 @@ ApiRepo.prototype._buildProtos =
         this._findProtos(name, version, findOutputs, copyJavaPb);
       } else if (language === 'nodejs') {
         /**
+         * makeProtoBasedNodeModule finds the required .proto files for the API
+         * and copies them into the output directory.
+         */
+        var makeProtoBasedNodeModule =
+            function makeProtoBasedNodeModule(fullPathProtos, includePath) {
+          /** Add an entry for filePath to filesMap. */
+          function addToFilesMap(filePath) {
+            if (filePath in filesMap) {
```

Comment:
Fixed.  I was thinking jshint raised warning in the original order -- and then it was wrong, it's okay to be placed the variable before that.